### PR TITLE
Document caveat with SDK package URLs

### PR DESCRIPTION
**Pull request by pothos**
See https://github.com/flatcar-linux/scripts/pull/80

### CODE REVIEW

File: os/sdk-modifying-flatcar.md [hunk 1]
```diff
@@ -114,19 +114,21 @@ to switch to a recent nightly version because it only dereferences the nightly o
 build environment reproducible.
 
 ```
-./set_version --dev-board --board-version amd64-usr/CHANNEL-nightly
+./set_version --dev-board --board-version amd64-usr/CHANNEL-nightly --no-dev-sdk
+# --no-dev-sdk needed because --dev-board implies --dev-sdk
 ```
 
 To switch to a Flatcar release use the following:
```
Comment (marga-kinvolk) [inline os/sdk-modifying-flatcar.md:121]: Can you add a bit more text here explaining why someone would want to switch?

File: os/sdk-modifying-flatcar.md [hunk 2]
```diff
@@ -114,19 +114,21 @@ to switch to a recent nightly version because it only dereferences the nightly o
...
 
 ```
-./set_version --no-dev-board --board-version x.y.z  # --no-dev-board disables any previous --dev-board setting
+./set_version --no-dev-board --board-version x.y.z
+# --no-dev-board disables any previous --dev-board setting
 ```
 
 You should also use this command to switch to a newer SDK release if available.
```
Comment (marga-kinvolk) [inline os/sdk-modifying-flatcar.md:128]: I find the "should" here confusing... Shouldn't this be "can"? As in: "You can also use this command"... ?

Comment (pothos) [reply os/sdk-modifying-flatcar.md:128]: Thanks, changed and added _so that you don't have to rebuild SDK packages when you checked out more recent branches._

File: os/sdk-modifying-flatcar.md (hunk 1 above)
Comment (pothos) [reply os/sdk-modifying-flatcar.md:121]: Thanks, added _when you checked out release branches_ – does this help?

**Pull request merged**